prow/cmd/deck/main.go

```diff
@@ -94,6 +94,7 @@ func main() {
 	mux.Handle("/log", gziphandler.GzipHandler(handleLog(ja)))
 	mux.Handle("/rerun", gziphandler.GzipHandler(handleRerun(kc)))
 	mux.Handle("/config", gziphandler.GzipHandler(handleConfig(configAgent)))
+	mux.Handle("/config.js", gziphandler.GzipHandler(handleJSONConfig(configAgent)))
```

Comment:
we should probably make an endpoint to serve just the deck config, [the existing endpoint](https://prow.k8s.io/config) serves 777KB of prow config on our deployment ATM.

Reply:
:this_is_fine:

Reply:
updated to include only the branding config.